xtask/src/ci.rs

```diff
@@ -197,6 +198,13 @@ fn run_appservice_tests() -> Result<()> {
 }
 
 fn run_wasm_checks(cmd: Option<WasmFeatureSet>) -> Result<()> {
+    // it's the alias
```

Comment:
I find this comment much less clear than the code it applies to. Maybe remove it?